examples-esp32c3/Cargo.toml

```diff
@@ -25,6 +25,9 @@ esp-println = { workspace = true, features = [ "esp32c3" ] }
 esp-backtrace = { workspace = true, features = ["esp32c3"] }
 esp-wifi = { path = "../esp-wifi", features = ["esp32c3"] }
 
+[profile.dev]
```

Comment:
this is part of a workspace and this won't have any effect, same for `examples-esp32c6/Cargo.toml`

Reply:
Oh thanks. Looks like i accidentally left this in the PR. Will be removed later.